source/deprecations/v3.x.html.md

```diff
@@ -53,3 +53,57 @@ object.notifyPropertyChange('someProperty');
 ##### id: ember-metal.getting-each
 
 Calling `array.get('@each')` is deprecated. `@each` may only be used as dependency key.
+
+### Deprecations Added in Upcoming Versions
+
+#### Use console rather than Ember.Logger
+
+##### until: 4.0.0
+##### id: ember-console.deprecate-logger
+
+The `Ember.Logger` module will be removed at the start of the 4.x cycle. You should remove any calls to `Ember.Logger` and use calls to `console` instead.
```

Comment:
Directly under this line, can you show a few before and after examples?

I'm thinking something like:


```js
// Before
Ember.Logger.log('my little pony');

// After
console.log('my little pony');
```

Reply:
Added. While I was at it, I combined the before/after pairs down on the other examples.  Hopefully that doesn't violate editorial conventions too badly. They do read better with the code lines closer together and, with three of them, it saves significant space and plays down the changes a little.